portfolio/src/main/webapp/style.css

```diff
@@ -13,8 +13,49 @@
   margin-top: 20px;
 }
 
+/* Nav bar styling */
+
 .nav-bar {
   list-style-type: none;
   margin: 0;
   padding: 0;
+}
+
+.nav-bar li {
+  display: inline;
+}
+
+/* Gallery styling */
```

Comment:
nit: maybe add a new line after this to be consistent with the other /* nav bar styling */ ?